tests/lib/rules/no-only-experiments.js

```diff
@@ -22,8 +22,11 @@ ruleTester.run("no-only-experiments", rule, {
 
     valid: [
         'lab.experiment("valid", function() {})',
+        'lab.test("valid", function() {})',
         'lab.experiment',
+        'lab.test',
         'experiment("valid", function() {})',
+        'lab("valid", function() {})',
```

Comment:
You should also add a case for `lab.it('valid', function() {})` and `lab.it`

Reply:
Okay can do! They didn't add a case for describe so I just followed what they had done. So I should also add a case for describe as well.

Reply:
Would there need to be a test case for 'it("valid", function() {})' ?

Reply:
That seems like a good idea